components/match2/commons/starcraft_starcraft2/match_group_input_starcraft.lua

```diff
@@ -386,7 +381,11 @@ function StarcraftMatchGroupInput.SubMatchStructure(match)
 				winner = 0
 			}
 			--adjust sub-match scores
-			if tonumber(match['map' .. i].winner) == 1 then
+			if match['map' .. i].map and String.startsWith(match['map' .. i].map, 'Submatch') then
+				for opponentIx, score in ipairs(SubMatches[k].scores) do
```

Comment:
`k` is not a very descriptive variable name
(Separate PR)

Reply:
yeah i am already working on a general clean up at https://liquipedia.net/commons/Module:MatchGroup/Input/Starcraft/cleanup